### PR TITLE
Update to Compose alpha04, Canary 13

### PR DESCRIPTION
**Pull request by jimKarsh**
Updated alignments for alpha04 
Updated compose workaround to reference new compose compiler location

### CODE REVIEW

File: sample/todo/list/src/androidMain/kotlin/com/arkivanov/todo/list/TodoListComposable.kt [hunk 1]
```diff
@@ -20,7 +19,7 @@ import com.arkivanov.decompose.invoke
 @Composable
 operator fun TodoList.Model.invoke() {
     data { data ->
-        LazyColumnFor(items = data.items, modifier = Modifier.weight(1F)) { item ->
```
Comment (arkivanov) [inline sample/todo/list/src/androidMain/kotlin/com/arkivanov/todo/list/TodoListComposable.kt:23]: So it is now applied by default, right?

Comment (jimKarsh) [reply sample/todo/list/src/androidMain/kotlin/com/arkivanov/todo/list/TodoListComposable.kt:23]: I can't find defaults in the documentation regarding weight / width; however, according to a google dev Adam Powell in #compose channel, the weight modifier is ignored in this scope. After removing it the list is still full screen width. He also stated the compose team is not happy with the current API for column/row scopes so, in his words, "expect this to change again".

**Review by arkivanov — APPROVED** [review]

File: sample/todo/list/src/androidMain/kotlin/com/arkivanov/todo/list/TodoListComposable.kt (hunk 1 above)
Comment (arkivanov) [reply sample/todo/list/src/androidMain/kotlin/com/arkivanov/todo/list/TodoListComposable.kt:23]: Makes total sense, thanks!

**Comment by arkivanov** [comment]
Thanks for your contribution!

**Pull request merged**